### PR TITLE
 announcing International PHP Conference Munich 2022

### PR DESCRIPTION
**Pull request by rmunsky**
Hi

it would be great if you could merge this pull request.

Thanks in advance,

Robert

### CODE REVIEW

File: archive/entries/2018-10-12-1.xml [hunk 1]
```diff
@@ -20,4 +20,4 @@
 
     </div>
   </content>
-</entry>
+</entry>
```
Comment (cmb69) [inline archive/entries/2018-10-12-1.xml:23]: Please revert this modification; this file is unrelated.

**Review by cmb69 — COMMENTED** [review]
Thank you for the PR! Looks good, except for the modified unrelated file.

File: archive/entries/2018-10-12-1.xml (hunk 1 above)
Comment (ajelodar) [reply archive/entries/2018-10-12-1.xml]: Hi, I revert the changes. 
Could you please merge the pull request?

Thanks in advance,
Amir

File: archive/entries/2022-07-06-1.xml [hunk 2]
```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0" encoding="utf-8"?>
+<entry xmlns="http://www.w3.org/2005/Atom" xmlns:default="http://php.net/ns/news">
+  <title>International PHP Conference Munich 2022</title>
+  <id>http://php.net/archive/2021.php#id2022-07-06-1</id>
```
Comment (cmb69) [inline archive/entries/2022-07-06-1.xml:4]:
```suggestion
  <id>http://php.net/archive/2021.php#2022-07-06-1</id>
```

**Comment by cmb69** [comment]
Thank you!
